crm_compassion/models/event_compassion.py

```diff
@@ -261,6 +269,11 @@ def write(self, vals):
         """ Push values to linked objects. """
         super(event_compassion, self).write(vals)
 
+        if self.hold_start_date > self.start_date:

```

Comment:
Instead of checking in the write method, use a `@api.constrains` method.
See https://www.odoo.com/documentation/8.0/reference/orm.html#module-openerp.api